docs/user/performance.rst

```diff
@@ -13,80 +13,58 @@ Use magnitudes when possible
+    214 ns +- 2.39 ns per loop (mean +- std. dev. of 7 runs, 1,000,000 loops each)
 
 This is especially important when using pint Quantities in conjunction with an iterative solver, such as the `brentq method`_ from scipy:
 
-.. doctest::
+.. ipython::
+    :verbatim:
 
     In [1]: from scipy.optimize import brentq
 
     In [2]: def foobar_with_quantity(x):
-                # find the value of x that equals q2
-
-                # assign x the same units as q2
-                qx = ureg(str(x)+str(q2.units))
-
-                # compare the two quantities, then take their magnitude because
-                # brentq requires a dimensionless return type
-                return (qx - q2).magnitude
+       ...:     # find the value of x that equals q2
+       ...:
+       ...:     # assign x the same units as q2
+       ...:     qx = ureg(str(x)+str(q2.units))
+       ...:
+       ...:     # compare the two quantities, then take their magnitude because
+       ...:     # brentq requires a dimensionless return type
+       ...:     return (qx - q2).magnitude
 
     In [3]: def foobar_with_magnitude(x):
-                # find the value of x that equals q2
-
-                # don't bother converting x to a quantity, just compare it with q2's magnitude
-                return x - q2.magnitude
+       ...:     # find the value of x that equals q2
+       ...:
+       ...:     # don't bother converting x to a quantity, just compare it with q2's magnitude
+       ...:     return x - q2.magnitude
 
     In [4]: %timeit brentq(foobar_with_quantity,0,q2.magnitude)
-    1000 loops, best of 3: 310 µs per loop
+    286 us +- 9.05 us per loop (mean +- std. dev. of 7 runs, 1,000 loops each)
 
     In [5]: %timeit brentq(foobar_with_magnitude,0,q2.magnitude)
-    1000000 loops, best of 3: 1.63 µs per loop
+    1.14 us +- 21.3 ns per loop (mean +- std. dev. of 7 runs, 1,000,000 loops each)
 
 Bear in mind that altering computations like this **loses the benefits of automatic unit conversion**, so use with care.
 
 A safer method: wrapping
 ------------------------
 A better way to use magnitudes is to use pint's wraps decorator (See :ref:`wrapping`). By decorating a function with wraps, you pass only the magnitude of an argument to the function body according to units you specify. As such this method is safer in that you are sure the magnitude is supplied in the correct units.
 
-.. doctest::
-
-    In [1]: import pint
-
-    In [2]: ureg = pint.UnitRegistry()
-
-    In [3]: import numpy as np
-
-    In [4]: def f(x, y):
-                  return (x - y) / (x + y) * np.log(x/y)
-
-    In [5]: @ureg.wraps(None, ('meter', 'meter'))
-             def g(x, y):
-                 return (x - y) / (x + y) * np.log(x/y)
-
-    In [6]: a = 1 * ureg.meter
-
-    In [7]: b = 1 * ureg.centimeter
-
-    In [8]: %timeit f(a, b)
-    1000 loops, best of 3: 312 µs per loop
-
-    In [9]: %timeit g(a, b)
-    10000 loops, best of 3: 65.4 µs per loop
```

Comment:
is there a specific reason why you removed this? The example seems important?

Reply:
I think this deletion was not wanted I'll put it back

Reply:
I've put back this part, there was a mistake putting this as doctest since it should be under ipython directive